diff --git a/dist/index.js b/dist/index.js
index 4d7c5582a97e2b3c5e51de433677cef5dfadd28a..5164093e9e757b5bfb795d492663dc47cc498b24 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,107 +1,107 @@
 "use strict";
 var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
-    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
-    return new (P || (P = Promise))(function (resolve, reject) {
-        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
-        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
-        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
-        step((generator = generator.apply(thisArg, _arguments || [])).next());
-    });
+  function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
+  return new (P || (P = Promise))(function (resolve, reject) {
+    function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+    function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+    function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+    step((generator = generator.apply(thisArg, _arguments || [])).next());
+  });
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.killAfterCleanup = exports.exitAfterCleanup = exports.removeCleanupListener = exports.addCleanupListener = void 0;
 let cleanupListeners;
 /** Registers a new cleanup listener. Adding the same listener more than once has no effect. */
 function addCleanupListener(listener) {
-    // Install exit listeners on initial cleanup listener
-    if (!cleanupListeners) {
-        installExitListeners();
-        cleanupListeners = new Set();
-    }
-    cleanupListeners.add(listener);
+  // Install exit listeners on initial cleanup listener
+  if (!cleanupListeners) {
+    installExitListeners();
+    cleanupListeners = new Set();
+  }
+  cleanupListeners.add(listener);
 }
 exports.addCleanupListener = addCleanupListener;
 /** Removes an existing cleanup listener, and returns whether the listener was registered. */
 function removeCleanupListener(listener) {
-    return cleanupListeners != null && cleanupListeners.delete(listener);
+  return cleanupListeners != null && cleanupListeners.delete(listener);
 }
 exports.removeCleanupListener = removeCleanupListener;
 /** Executes all cleanup listeners and then exits the process. Call this instead of `process.exit` to ensure all listeners are fully executed. */
 function exitAfterCleanup(code = 0) {
-    return __awaiter(this, void 0, void 0, function* () {
-        yield executeCleanupListeners();
-        process.exit(code);
-    });
+  return __awaiter(this, void 0, void 0, function* () {
+    yield executeCleanupListeners();
+    process.exit(code);
+  });
 }
 exports.exitAfterCleanup = exitAfterCleanup;
 /** Executes all cleanup listeners and then kills the process with the given signal. */
 function killAfterCleanup(signal) {
-    return __awaiter(this, void 0, void 0, function* () {
-        yield executeCleanupListeners();
-        process.kill(process.pid, signal);
-    });
+  return __awaiter(this, void 0, void 0, function* () {
+    yield executeCleanupListeners();
+    process.kill(process.pid, signal);
+  });
 }
 exports.killAfterCleanup = killAfterCleanup;
 function executeCleanupListeners() {
-    return __awaiter(this, void 0, void 0, function* () {
-        if (cleanupListeners) {
-            // Remove exit listeners to restore normal event handling
-            uninstallExitListeners();
-            // Clear cleanup listeners to reset state for testing
-            const listeners = cleanupListeners;
-            cleanupListeners = undefined;
-            // Call listeners in order added with async listeners running concurrently
-            const promises = [];
-            for (const listener of listeners) {
-                try {
-                    const promise = listener();
-                    if (promise)
-                        promises.push(promise);
-                }
-                catch (err) {
-                    console.error("Uncaught exception during cleanup", err);
-                }
-            }
-            // Wait for all listeners to complete and log any rejections
-            const results = yield Promise.allSettled(promises);
-            for (const result of results) {
-                if (result.status === "rejected") {
-                    console.error("Unhandled rejection during cleanup", result.reason);
-                }
-            }
+  return __awaiter(this, void 0, void 0, function* () {
+    if (cleanupListeners) {
+      // Remove exit listeners to restore normal event handling
+      uninstallExitListeners();
+      // Clear cleanup listeners to reset state for testing
+      const listeners = cleanupListeners;
+      cleanupListeners = undefined;
+      // Call listeners in order added with async listeners running concurrently
+      const promises = [];
+      for (const listener of listeners) {
+        try {
+          const promise = listener();
+          if (promise)
+            promises.push(promise);
+        }
+        catch (err) {
+          console.error("Uncaught exception during cleanup", err);
         }
-    });
+      }
+      // Wait for all listeners to complete and log any rejections
+      const results = yield Promise.allSettled(promises);
+      for (const result of results) {
+        if (result.status === "rejected") {
+          console.error("Unhandled rejection during cleanup", result.reason);
+        }
+      }
+    }
+  });
 }
 function beforeExitListener(code) {
-    console.log(`Exiting with code ${code} due to empty event loop`);
-    void exitAfterCleanup(code);
+  // console.log(`Exiting with code ${code} due to empty event loop`);
+  void exitAfterCleanup(code);
 }
 function uncaughtExceptionListener(error) {
-    console.error("Exiting with code 1 due to uncaught exception", error);
-    void exitAfterCleanup(1);
+  // console.error("Exiting with code 1 due to uncaught exception", error);
+  void exitAfterCleanup(1);
 }
 function signalListener(signal) {
-    console.log(`Exiting due to signal ${signal}`);
-    void killAfterCleanup(signal);
+  // console.log(`Exiting due to signal ${signal}`);
+  void killAfterCleanup(signal);
 }
 // Listenable signals that terminate the process by default
 // (except SIGQUIT, which generates a core dump and should not trigger cleanup)
 // See https://nodejs.org/api/process.html#signal-events
 const listenedSignals = [
-    "SIGBREAK",
-    "SIGHUP",
-    "SIGINT",
-    "SIGTERM",
-    "SIGUSR2", // Used by Nodemon
+  "SIGBREAK",
+  "SIGHUP",
+  "SIGINT",
+  "SIGTERM",
+  "SIGUSR2", // Used by Nodemon
 ];
 function installExitListeners() {
-    process.on("beforeExit", beforeExitListener);
-    process.on("uncaughtException", uncaughtExceptionListener);
-    listenedSignals.forEach((signal) => process.on(signal, signalListener));
+  process.on("beforeExit", beforeExitListener);
+  process.on("uncaughtException", uncaughtExceptionListener);
+  listenedSignals.forEach((signal) => process.on(signal, signalListener));
 }
 function uninstallExitListeners() {
-    process.removeListener("beforeExit", beforeExitListener);
-    process.removeListener("uncaughtException", uncaughtExceptionListener);
-    listenedSignals.forEach((signal) => process.removeListener(signal, signalListener));
+  process.removeListener("beforeExit", beforeExitListener);
+  process.removeListener("uncaughtException", uncaughtExceptionListener);
+  listenedSignals.forEach((signal) => process.removeListener(signal, signalListener));
 }
 //# sourceMappingURL=index.js.map
